### DISCUSSION TITLE
Development Tree is currently at the most recent version

### DISCUSSION THREAD

**Discussion by MoonlitBeans** [jgoll7/Tafe22S211#13] [General]
I updated the Development Tree, and pushed it. The most recent version with all the changes is currently on GitHub. I haven't tested the Mortgage Calculator at this current time though, or whether we can go to the basic calculator and back to the main menu.

I can look at those though and let you know of any updates that need to be done.

Okay so just did some quick testing.

We can't access the simple calculator through the MainMenu.xaml.
On the MortgageCalculator, can you move the two buttons to the left and centre of the text boxes and blocks? I'm unable to select the buttons because they are cut off by my smaller screen. Also, it should just be "Months" under the Years block. It looks a bit cleaner than "and Months". 

:)

Edit: Oh, I see thats exactly what the preview picture had. Nevermind the changes on the Mortgage Calculator. I can fix the issue with the MainMenu.xaml for the basic calculator though. I did fix the simple calculator issue and pushed it to the Repos. It is now fully updated for the time being.

**Comment by jgoll7** [comment]
Are we able to find a way to go back from the basic calculator to the main menu? Should I add an exit button to that page?

Also I think I will need to make the exit button on the main menu work, so that it will close the application.

**Reply by MoonlitBeans** [reply]
> Are we able to find a way back from the basic calculator to the main menu?

I thought about that. I think we should but I don't know how that would look right since there's an even number of buttons at the moment.

> Also I think I will need to make the exit button menu work

Good idea. I didn't even consider that tbh.

**Reply by jgoll7** [reply]
Ok I'll look into making both of these happen in the development branch, before working on the release branch.
